### PR TITLE
cnf ran: reorganize powermanagement helpers

### PR DESCRIPTION
**Pull request by klaskosk**
This moves the powermanagement stats file to be a package under ran and adds unit tests for it. Additionally, the metric collections helpers have been moved to their own package under powermanagement.

### CODE REVIEW

**Review by yliu127 — APPROVED** [review]

File: tests/cnf/ran/powermanagement/internal/collect/collect.go [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	if len(stressNgPods) < 1 {
+		return nil, fmt.Errorf("not enough stress-ng pods to run test")
+	}
+
+	glog.V(tsparams.LogLevel).Infof("Wait for %s for steadyworkload scenario", duration.String())
+	result, collectErr := collectPowerUsageMetrics(duration, interval, "steadyworkload", tag)
```
Comment (sebrandon1) [inline tests/cnf/ran/powermanagement/internal/collect/collect.go:76]: Should `collectErr` be checked immediately after this?

**Review by sebrandon1 — APPROVED** [review]
A comment about an error check.

File: tests/cnf/ran/powermanagement/internal/collect/collect.go (hunk 1 above)
Comment (klaskosk) [reply tests/cnf/ran/powermanagement/internal/collect/collect.go:76]: I think this came up when the test got merged the first time lol but my idea is that we want to tear down the stress pods regardless of whether the metrics collection failed. Might be better to delete the pods in a `defer` but the preference seems to be against `defer` in tests

Comment (sebrandon1) [reply tests/cnf/ran/powermanagement/internal/collect/collect.go:76]: I think `defer` is fine in tests just outside of a loop.

Sounds good though!

**Pull request merged**